full_snapshot.proto

```diff
@@ -235,6 +236,67 @@ message TablespaceInformation {
   repeated string config = 4;   // Tablespace-level options, as "keyword=value" strings
 }
 
+message CollectorConfig {
```

Comment:
@lfittl so I added everything from Config except things that could obviously have credentials like URLs and passwords  and things that are vanishingly likely to be useful in the app like cert contents. We may want to drop some other things. The numbering is a little sloppy right now since I expect we'll renumber after we remove (or add) fields. I was also wondering if we should skip numbers in case we want to expand a section (e.g., more azure-specific settings). Thoughts?

Reply:
Re: numbering - maybe we start a fresh block of 10 fields for every provider? (so we have a few fields free)